ci/axis/rapids-core-devel.yaml

```diff
@@ -0,0 +1,33 @@
+BUILD_IMAGE:
+  - rapidsai/rapids-core-dev
+  - rapidsai/rapids-core-dev-nightly
```

Comment:
these should be `rapidsai/rapidsai-core-dev` and `rapidsai/rapidsai-core-nightly-dev`

and update the file name and exclusions